src/ucp/core/ucp_worker.h

```diff
@@ -290,9 +294,9 @@ typedef struct ucp_worker {
 
     ucp_worker_rkey_config_hash_t    rkey_config_hash;    /* RKEY config key -> index */
     ucp_worker_discard_uct_ep_hash_t discard_uct_ep_hash; /* Hash of discarded UCT EPs */
-    ucs_ptr_map_t                    ep_map; /* UCP ep key to ptr mapping */
-    ucs_ptr_map_t                    request_map; /* UCP requests key to ptr
-                                                     mapping */
+    UCS_PTR_MAP_T(ep) ep_map; /* UCP ep key to ptr mapping */
+    UCS_PTR_MAP_T(request)
```

Comment:
pls move the name to the same string

Reply:
Ok. This change was suggested by `format code` pipeline stage.